scripts/generate_export.py

```diff
@@ -451,11 +455,28 @@ def sequences_overlap(true_seq, pred_seq):
             ].apply(lambda x: x.ffill().bfill())
             output_df = output_df[col_order]
 
+            for cs in complete_submissions:
+                complete_filenames.append(str(cs.input_filename))
+
+            complete_filenames_df = pd.DataFrame(complete_filenames, columns=["filename"])
+        
+            for cs in complete_submissions:
+                sub_job = (IndicoClient.call(SubmissionResult(cs.id, wait=True)))
+                result = (IndicoClient.call(RetrieveStorageObject(sub_job.result)))
+                complete_revID.append(result.get('reviewer_id'))
+    
+            complete_revID_df = pd.DataFrame(complete_revID, columns=["Reviewer ID"])
+        
+            reviewer_filename_df = pd.concat([complete_filenames_df, complete_revID_df], axis=1)
```

Comment:
I notice you do this below in the exceptions portion. We should probably make this a function.